### ISSUE TITLE
Error pid file. 

### ISSUE THREAD

**Issue by laurent6** [citusdata/pg_auto_failover#606]
I've problem with pg_autoctl create monitor. 
```
11:57:22 8950 INFO   /usr/lib/postgresql/13/bin/postgres -D /var/lib/postgresql/monitor -p 5000 -h *
11:57:32 8940 ERROR Failed to open file "/var/lib/postgresql/monitor/postmaster.pid": No such file or directory
11:57:32 8941 ERROR Failed to open file "/var/lib/postgresql/monitor/postmaster.pid": No such file or directory
11:57:32 8940 INFO  Is PostgreSQL at "/var/lib/postgresql/monitor" up and running?
11:57:32 8941 INFO  Is PostgreSQL at "/var/lib/postgresql/monitor" up and running?
11:57:32 8940 ERROR Failed to get Postgres pid, see above for details
11:57:32 8941 ERROR Failed to get Postgres pid, see above for details
``` 
postgesql13 is running. What do I have forget ?

**Comment by DimCitus** [comment]
Hi @laurent6 ; thanks for reporting a bug (that's the only way we know how to fix them, after all...). Could you show the command you issued, and run it again with `-vv` (very verbose output) so that we have more information about your situation here? Thanks.

**Comment by laurent6** [comment]
I've solved it
I installed apt-get install postgresql-11-auto-failover instead of apt-get install postgresql-13-auto-failover. 
->For this issues apt purge  ( all postgresql server  and faillover ) and install it again. 
We can't run it in root mode. Connect postgres instead of root. 
Make sure that change pgdata in  absolute and not relative path. ( i'm not sure, maybe my problem came from there ) 
change right /var/run/suer/0 ( default postgres can't update it )
